torch_xla_py/xla_dist.py

```diff
@@ -53,8 +72,34 @@ def __hash__(self):
 
 
 class ServiceWorker(Worker):
-  # Same as base Worker ATM.
-  pass
+
+  def __init__(self, internal_ip, port, machine_type, zone, sw_version):
+    super(ServiceWorker, self).__init__(internal_ip, machine_type, zone)
+    self._port = str(port)
```

Comment:
Here, you can eventually do an `int()` and enforce that a port must be integer.

Reply:
Done.